contracts/contracts/strategies/BaseConvexMetaStrategy.sol

```diff
@@ -160,6 +163,22 @@ abstract contract BaseConvexMetaStrategy is BaseCurveStrategy {
         revert("Invalid Metapool asset");
     }
 
+    /**
+     * @dev Sets max withdrawal slippage that is considered when removing
```

Comment:
nit: document what unit is used for _maxWithdrawalSlippage. It's a percentage and looks like 100% = 1e18, 1% = 1e16

Reply:
yup good point, adding that